fineract-provider/src/main/java/org/apache/fineract/portfolio/loanproduct/serialization/LoanProductDataValidator.java

```diff
@@ -278,7 +278,7 @@ public void validateForCreate(final String json) {
             final Long delinquencyBucketId = this.fromApiJsonHelper.extractLongNamed(LoanProductConstants.DELINQUENCY_BUCKET_PARAM_NAME,
                     element);
             baseDataValidator.reset().parameter(LoanProductConstants.DELINQUENCY_BUCKET_PARAM_NAME).value(delinquencyBucketId)
-                    .ignoreIfNull().integerGreaterThanZero();
+                    .integerGreaterThanZero();
```

Comment:
integerGreaterThanZero will not throw error if the value is null, but it shall fail... Please add "notNull()"

Reply:
Done!